src/sdk/makeChart.js

```diff
@@ -179,9 +179,20 @@ export default ({
     })
 
     if (wasLoaded && !errored) node.trigger("successFetch", nextPayload, prevPayload)
+    invalidateContexts(nextRawPayload?.versions?.contexts_hard_hash)
     finishFetch()
   }
 
+  const invalidateContexts = hardHash => {
+    if (!hardHash) return
+    const container = node.getParent()
+    if (!container) return
```

Comment:
nit: in general let's try to keep function parts separate with a new line every now and then 😄 
```
const invalidateContexts = hardHash => {
    if (!node || !hardHash) return

    const container = node.getParent()
    if (!container) return

    const containerHardHash = container.getAttribute("contextsHardHash")

    if (containerHardHash !== hardHash)
      container.updateAttribute("contextsHardHash", hardHash)
  }
```